docs/blockchainwallets.md

```diff
@@ -0,0 +1,95 @@
+# Blockchain Wallets
+
+# What is a crypto wallet?
+
+In essence, a cryptocurrency wallet is a gadget that allows you to communicate with a blockchain network. There are various types of cryptocurrency wallets, which can be divided into three categories: software, hardware, and paper wallets. Depending on how they work, they may alternatively be referred to as "hot" or "cold" wallets.
+
+Because of this, most software-based crypto wallet providers are more user-friendly than hardware wallets. Hardware wallets are the most secure choice. Paper wallets, on the other hand, are merely a sheet of paper with a printed-out "wallet" on it.
+
+# How do cryptocurrency wallets work?
+
+Contrary to a popular misconception, crypto wallets do not actually hold digital assets. They instead supply the tools needed to communicate with a blockchain. To put it another way, these wallets can create the information required to transfer and receive money via blockchain transactions. Such information includes, among other things, one or more pairs of public and private keys.
+
+The wallet also contains an address, which is an alphanumeric identifier formed from the public and private keys. In essence, an address is a distinct "location" on the blockchain to which bitcoin can be transmitted. This implies that while you can share your address with others in order to receive payments, you should never reveal your private key with anyone.
+
+Regardless of whose wallet you use, the private key grants access to your cryptocurrency. As a result, even if your computer or smartphone is compromised, you can still access your assets on another device if you have the associated private key (or seed phrase). It is important to note that the coins are never completely removed from the blockchain; they are simply transferred from one address to another.
+
+# Types of wallets
+
+## Hardware wallets

```

Comment:
Might be useful to add a couple of links where users could buy a hardware wallet?
Is it a good/bad idea to add affiliate links?